modular-docs-manual/content/topics/module_guidelines-assembly.adoc

```diff
@@ -27,7 +27,7 @@ Use the second level heading syntax for the Prerequisites section in the assembl
 [discrete]
 == Assembly Modules
 
-List link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#include-files[include files] to include the required modules. Use any combination of concept, procedure, and reference modules that fulfills the purpose of the assembly.
+List link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#include-files[include files] to include the required modules. Use any combination of concept, procedure, and reference modules that fulfills the purpose of the assembly. A `leveloffset` can provide a method of introducing hierarchy, such as `module-file.adoc[leveloffset=+2]`.
```

Comment:
```suggestion
List link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#include-files[include files] to include the required modules. Use any combination of concept, procedure, and reference modules that fulfills the purpose of the assembly. You can use a `leveloffset` attribute, such as `module-file.adoc[leveloffset=+1]`,  to set the hierarchy of the module relative to the assembly.
```
Reworded this to speak more directly to the writer's needs.

Reply:
@rolfedh Okay on bullet one. On bullet two, that's kind of the point of this note, to specify that leveloffset of 2 is fine to use when needed. There are many cases when a module is a subchapter after another module, such as a high-level concept module that has relevant proc modules that are related to it. There is a misnomer in the past and maybe still out there that leveloffset of 2 or 3 is forbidden, which isn't true.